### PR TITLE
Adding dependabot

### PR DESCRIPTION
**Pull request by rcolfin**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+      interval: "weekly"
+    commit-message:
+      prefix: "github action updates"
+
+  - package-ecosystem: pip
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    commit-message:
+      prefix: "pip updates"
```
Comment (GitBib) [inline .github/dependabot.yml:16]: Please note that pymkv2 uses Poetry for dependency management, so using pip updates might not help you get the latest updates.

Comment (rcolfin) [reply .github/dependabot.yml:16]: Yeah.  I was looking at the documentation for [dependabot](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file) and it says to indicate pip even for Poetry.  I'm using this in another one of my repos.

The biggest pain I had with it was it creating multiple PRs to upgrade the dependencies.  I believe there is a way to group them though.  I am also exploring in one of my repos to auto merge if from dependabot.

Comment (rcolfin) [reply .github/dependabot.yml:16]: Updated the PR to also add dependabotautomerge.yml to automatically merge the dependabot changes (provided the build succeeds on the other checks)

**Comment by rcolfin** [comment]
@GitBib Let me know if you need any other changes to this.

**Pull request merged**